### DISCUSSION TITLE
Welcome to pytraction paper Discussions!

### DISCUSSION THREAD

**Discussion by latent-reason** [rg314/pytraction#38] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by latent-reason** [comment]
@AkiStubb, @andimi, @wolframponisch if you could post papers here that would be great :)

**Comment by andrea-dimitracopoulos** [comment]
Thanks for opening this @rg314! 

I would like to add to this discussion a few points that came up yesterday, which mainly relate to clarifying the steps of the algorithm used in pytraction:

- How does pytraction choose the best window size for the iterative PIV, and what should the user consider when explicitly setting it?
- What algorithm is used for the Baysian estimation of the regularisation parameter?
- Difference between estimating one regularisation parameter for the whole stack/dataset vs a single frame?
- What part of the image is used from the frame(s) when the ROI is not specified, and which one when it is specified?

I think there were more points related to the algorithm, simulations, and experimental data that we can add here if @rg314 you think it's useful?
